plugins/copr.py

```diff
@@ -475,28 +477,40 @@ def _download_repo(self, project_name, repo_filename):
         api_path = "/coprs/{0}/repo/{1}/dnf.repo?arch={2}".format(project_name, short_chroot, arch)
 
         try:
-            f = self.base.urlopen(self.copr_url + api_path, mode='w+')
-        except IOError as e:
+            response = urlopen(self.copr_url + api_path)
             if os.path.exists(repo_filename):
                 os.remove(repo_filename)
-            if '404' in str(e):
-                try:
-                    res = urlopen(self.copr_url + "/coprs/" + project_name)
-                    status_code = res.getcode()
-                except HTTPError as e:
-                    status_code = e.getcode()
-                if str(status_code) != '404':
-                    raise dnf.exceptions.Error(_("This repository does not have"
-                                                 " any builds yet so you cannot enable it now."))
-                else:
-                    raise dnf.exceptions.Error(_("Such repository does not exist."))
-            raise
-
-        for line in f:
-            if re.match(r"\[copr:", line):
-                repo_filename = os.path.join(self.base.conf.get_reposdir,
-                                             "_" + line[1:-2] + ".repo")
-            break
+        except HTTPError as e:
+            if e.code != 404:
+                error_msg = _("Request to {0} failed: {1} - {2}".format(self.copr_url + api_path, e.code, str(e)))
+                raise dnf.exceptions.Error(error_msg)
+            error_msg = _("It wasn't possible to enable this project.\n")
```

Comment:
I would just delete this line, I find it redundant. But if you decide to keep it, note that on line 503 you overwrite the variable instead of appending, so this sentence is only printed for the `if` branch starting on line 489, which is confusing, if that's desired, you should put it inside the `if`.